wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/DefaultFactoryParameterResolver.java

```diff
@@ -31,14 +35,16 @@
 public class DefaultFactoryParameterResolver implements FactoryParametersResolver {
 
   public static final String LOCAL_FILES_REFERENCES_ARE_NOT_SUPPORTED =
-      "Devfile requires local file content but this functionality is not supported when URL to raw content is specified. "
-          + "You can specify URL to Github repository instead.";
+      "Devfile requires local file content but the remote location of the devfile is not known.";
```

Comment:
It would be better UX if message provides information on how to specify devfile remote location (not sure if it is possible here).